GoToProject/src/library/methods/auth.js

```diff
@@ -24,30 +24,29 @@ export const login = async (email, password, navigation, setStatus) => {
     });
 };
 
-export const register = async (email, password, username,latitude,length ) => {
+export const register = async (email, password, username ) => {
   return await auth()
     .createUserWithEmailAndPassword(email, password)
     .then(({user}) => {     
       user
         .updateProfile({displayName: username})
-        .then(() => createAditionalData(latitude, length));
+        .then(() => createUserData());
     })
     .catch(error => {
-      console.log(error)
+      Alert.alert('Sorry something went wrong :', error);
     });
 };
 
-const createAditionalData = (latitude, length) => {
+const createUserData = () => {
   firestore()
     .collection('users')
     .doc(auth().currentUser.uid)
     .get()
     .then(response => {
       if (!response.exists) {
+        console.log("Response exists");
```

Comment:
Recommendation: Do not upload the console.log() to GitHub, instead use the Alert element.

Reply:
Logs removed. They were needed only for testing purpose, I just forgot to remove them before uploading.